File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        StringBuilder answer = new StringBuilder();
        
        int length = phone_number.length() - 4;
        for (int i = 0; i < phone_number.length(); i++) {
            if (i < length)
                answer.append('*');
            else answer.append(phone_number.charAt(i));
        }
        
        return answer.toString();
    }
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.js
function solution(phone_number) {
    var answer = '';
    
    const size = phone_number.length;
    for (var i = 0; i < size; i++) {
        if (i < size-4)
            answer += "*";
        else answer += phone_number[i];
    }
    
    return answer;
}